src/main/resources/db/migration/V6_0__update_behandler_herid.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE BEHANDLER SET her_id='105186',updated_at=now() WHERE ID=20383 AND kontor_id=868;
+UPDATE BEHANDLER SET her_id='180028',updated_at=now() WHERE ID=33882 AND kontor_id=868;
+UPDATE BEHANDLER SET her_id='175449',updated_at=now() WHERE ID=25608 AND kontor_id=868;
+UPDATE BEHANDLER SET her_id='176644',updated_at=now() WHERE ID=28568 AND kontor_id=868;
+UPDATE BEHANDLER SET her_id='180064',updated_at=now() WHERE ID=33633 AND kontor_id=868;
+UPDATE BEHANDLER SET her_id='180006',updated_at=now() WHERE ID=33766 AND kontor_id=868;
+UPDATE BEHANDLER SET her_id='182585',updated_at=now() WHERE ID=37751 AND kontor_id=868;
```

Comment:
Jeg bare kopierte fra de andre scriptene, men hvorfor trenger man `AND kontor_id=` egentlig? `id` i tabellen er jo unik 🤷🏼‍♂️ Kanskje det er litt for syns skyld, og vise hva slags kontor vi oppdaterte på?
Leger kan jo være knyttet til flere kontorer, men ville trodd `id` uansett valgte riktig rad.